### DISCUSSION TITLE
2.7.1 更新日志

### DISCUSSION THREAD

**Discussion by LTCatt** [Meloong-Git/PCL#3645] [讨论]
由于 #3616 影响较大，临时发一个更新先把它修了，其他近期已处理的反馈会在下一次更新中实装。

· 修复了以下 2 个 Bug
　　更新新老版本的文件名相同的 Mod 会导致它被删除（#3606，@yangzimugreg）
　　启动部分整合包可能导致丢失游戏设置（#3616，@BOLTieKuai）

<hr /><em>This discussion was created from the release <a href='https://github.com/Hex-Dragon/PCL2/releases/tag/2.7.1'>2.7.1</a>.</em>

**Comment by wuliaodexiaoluo** [comment]
好耶！

**Comment by lucienshawls** [comment]
这个版本的EXE文件构建的时候似乎没改版本号？在详细信息里查看到的版本仍然是2.7.0.0
![image](https://github.com/Hex-Dragon/PCL2/assets/82591976/37bcd02e-af62-4bba-a9e0-8e78c1b97539)

这导致我打开PCL2并检查更新时，PCL2始终提示有新版本，而更新完了却还是检测到当前版本是2.7.0.0

不过考虑到这是个临时发的版本，这个bug就不发新issue了。

**Reply by wuliaodexiaoluo** [reply]
改了
认真看下关于与鸣谢，版本号是 2.7.0(319.50)
但是这段可能没写判断。。。。

**Comment by MoYuan-CN** [comment]
猫，你又忘记写更新日志了，就算只有 2 个 Bug（1 个重要）也还是要写的吧。。。
![image](https://github.com/Hex-Dragon/PCL2/assets/109946426/42b760e9-585b-4af0-a2af-5a18cd8168c1)

**Reply by wuliaodexiaoluo** [reply]
我把这个合在一个 Issue 了。。。
3647 帮我切下标签

**Reply by MoYuan-CN** [reply]
能不能不要把多个问题交在一个 Issue 中啊啊啊~~啊啊啊啊啊啊啊啊啊啊啊啊啊啊&nbsp;~~

**Reply by wuliaodexiaoluo** [reply]
按你的要求改了
3647 3648 切下 Label

**Comment by LTCatt** [comment]
关于该版本的问题见 #3651

**Comment by SqrtTan** [comment]
这就是为什么没有更新日志显示？
